bundles/framework/mydata/request/AddTabRequest.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
We should get rid of the request probably if we _don't_ try having a backwards compatible functionality that supports jQuery tabs. I don't think we should. Instead we should try to make myplaces etc external bundles pushing data to personaldata work with both the new "mydata" and the current "personaldata" bundles.